netman/adapters/switches/brocade.py

```diff
@@ -436,6 +448,14 @@ def _get_vlan(self, vlan_number, include_vif_data=False):
     def _show_vlan(self, vlan_number):
         return self.shell.do("show vlan {}".format(vlan_number))
 
+    def _get_operations(self, result):
```

Comment:
this could be a little clearer, how about _get_vlan_association_removal_operations ... just a suggestion